src/common/toast/hdsToast.tsx

```diff
@@ -41,10 +47,11 @@ const DismissableNotificationWrapper = ({
   );
 };
 
-const hdsToast = ({ type, labelText, text, toastId, translated = false }: HDSToastArgs) => {
+const hdsToast = ({ autoDismiss = false, type, labelText, text, toastId, translated = false }: HDSToastArgs) => {
```

Comment:
Why don't we default it to `true` if this is the desired behavior?

Reply:
Good thought, I'll flip this around